include/json/reader.h

```diff
@@ -324,6 +324,8 @@ class JSON_API CharReaderBuilder : public CharReader::Factory {
    * - `"allowSpecialFloats": false or true`
    *   - If true, special float values (NaNs and infinities) are allowed and
    *     their values are lossfree restorable.
+   * - `"skipBom": false or true`
+   *   - If true, if the input starts with the Unicode BOM, it is skipped.
```

Comment:
```suggestion
   *   - If true, if the input starts with the Unicode byte order mark, it is skipped.
```

Reply:
Updated it with `byte order mark (BOM)` if that's okay?

Reply:
ok